woocommerce/woocommerce-ios
# [Scan to Update Inventory M1] Flow when the stock is not managed

Part of: #11415 


## Description

With this PR we implement flow when the stock is not managed for a product or variation in the quick inventory update screen. The error notice will follow when we merge the error notices PR, so it can be reused.

## Testing instructions


With a product whose stock is not managed.

1. Go to products
2. Tap on the barcode scanner button
3. Scan the product barcode.
4. The quick inventory screen is open. See that we hide the quantity button and the we show a button to manage the stock.
5. Tap on that button. A loading indicator replaces the button.
6. When the operation finishes, the quantity appears (0) and the user can update the stock quantity.

Try also with a product variation.

## Screenshots


https://github.com/woocommerce/woocommerce-ios/assets/1864060/54373a36-babd-4ab8-ad69-ad2f324b479f



---
